storage/src/mmr/proof.rs

```diff
@@ -286,7 +284,7 @@ impl<D: Digest> Proof<D> {
         &self,
         hasher: &mut H,
         elements: &[E],
-        start_element_pos: u64,
+        start_loc: u64,
```

Comment:
It seems that `adb::verify::create_proof_store` (which calls `ProofStore::new`, which calls this function) still passes a _position_ rather than a _location_.